src/components/Script.jsx

```diff
@@ -48,7 +49,15 @@ const Script = ({ actors, currentScript }) => {
       />
       <h2>script</h2>
       <h5>Current Actor: {currentActor.name}</h5>
-      <div id="scriptDiv">{lineChunks}</div>
+      <div style={{display:'flex', width:'100%'}}
+      >
+        <div id="scriptDiv">
+          <div>
+            {lineChunks}
+          </div>
```

Comment:
Does lineChunks need to be wrapped in two divs?

Reply:
Yep, it's due to the current flex style modifications. Without it, each linechunk is centered: 
![image](https://user-images.githubusercontent.com/12701028/226723144-81440775-f020-48a0-8ef9-2cb382d03399.png)

Reply:
lol, that's a look. Thanks!

Reply:
The extra div is so _that_ div gets centered without everything within it rather than centering each line chunk